keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -80,9 +83,10 @@ class RetinaNet(keras.Model):
         bounding_box_format: The format of bounding boxes of input dataset. Refer
             [to the keras.io docs](https://keras.io/api/keras_cv/bounding_box/formats/)
             for more details on supported bounding box formats.
-        backbone: optional `keras.Model`. Must implement the `pyramid_level_inputs`
-            property with keys 3, 4, and 5 and layer names as values. If
-            `None`, defaults to `keras_cv.models.ResNet50V2Backbone()`.
+        backbone: `keras.Model`. Must implement the `pyramid_level_inputs`
+            property with keys 3, 4, and 5 and layer names as values. A somewhat
+            sensible backbone to use in many cases is the:
+            `keras_cv.models.ResNet50Backbone.from_preset('imagenet')`.
```

Comment:
This isn't a real preset. I'd recommend using the `keras_cv.models.ResNetBackbone.from_preset("resnet50_imagenet")` syntax.

Reply:
Fixed, thanks